### PR TITLE
Content migration batch 1

### PR DESCRIPTION
**Pull request by mattletford**
Migrating content

From: https://developers.google.com/web/updates/2015/11/devtools-digest-cds-edition
To: https://developers.google.com/blog/devtools-digest-cds-edition/

From: https://developers.google.com/web/updates/2015/10/chrome-47-webrtc
To: https://developers.google.com/blog/chrome-47-webrtc/

From: https://developers.google.com/web/updates/2015/07/improved-dom-breadcrumbs
To: https://developers.google.com/blog/improved-dom-breadcrumbs/

From: https://developers.google.com/web/updates/2015/04/a-new-home-for-devtools
To: https://developers.google.com/blog/a-new-home-for-devtools/

From: https://developers.google.com/web/updates/2015/05/toggle-the-devtools-dock-state-with-a-keyboard-shortcut
To: https://developers.google.com/blog/toggle-the-devtools-dock-state-with-a-keyboard-shortcut/

@rachelandrew

### CODE REVIEW

File: site/en/blog/devtools-digest-cds-edition/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+## A glimpse into the future of authoring
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/dJR-n8szgBc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+We’re just coming back from the Chrome Dev Summit where I showed you what working with DevTools looks like today and in the future. I usually don’t mention any features that are still experiments or heavy works of progress in this digest but I’m making an exception this time. If you don’t have time to watch the whole talk, here’s the gist:
+
+### Device Mode version 2
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/r-uc-fENa7w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (rachelandrew) [inline site/en/blog/devtools-digest-cds-edition/index.md:26]: These YouTube embeds need to use the shortcode https://developer.chrome.com/docs/handbook/components/#youtube

**Review by rachelandrew — COMMENTED** [review]
This looks fine other than the YouTube embed code which needs to use the shortcode - I linked to it inline.

**Review by rachelandrew — APPROVED** [review]

**Comment by rachelandrew** [comment]
@mattletford can you resolve the conflicts so I can merge this?

**Comment by mattletford** [comment]
> @mattletford can you resolve the conflicts so I can merge this?

@rachelandrew I've removed the tags to resolve the conflicts

**Pull request merged**